runner/task.go

```diff
@@ -182,13 +184,34 @@ func (t *Task) run(ctx Context, r *Run, s executionState) error {
 	return nil
 }
 
+// check if the command or any of the tasks in the stack are quiet
+func shouldBeQuiet(cmd Command, ctx Context) bool {
+	// never be quiet if the verbosity was increased
```

Comment:
While I don't personally do anything sensitive with tusk, my guess is that it would be more typical for a particular task to deal with secrets regularly, as opposed to a single task run to deal with secrets. That is to say, hiding secrets by changing the config file would be a more common use case than hiding secrets for a particular execution through CLI flags.

I'd lean toward, let's drop the check for verbose mode for now, and if it's a feature that folks end up asking for, it's definitely something we can revisit pretty easily.

Reply:
Removed this check.